package-build.el

```diff
@@ -516,7 +526,7 @@ Return a cons cell whose `car' is the root and whose `cdr' is the repository."
           ;; Using reset --hard here to comply with what's used for
           ;; unstable, but maybe this should be a checkout?
           (package-build--update-git-to-ref dir (concat "tags/" (cadr tag-version)))
-          ;; Return the version as a string
```

Comment:
I copied some of the Git logic for darcs, and this comment confused me for a moment before I remembered that it wasn't referring to the original version-string.